insertInterval.js

```diff
@@ -0,0 +1,46 @@
+var insert = function (intervals, newInterval) {
```

Comment:
https://leetcode.com/submissions/detail/879005017/

This pattern is very important for interviews -> Merge overlapping intervals